### PR TITLE
Enable yamllint and make the repo yamllintable

### PR DESCRIPTION
**Pull request by Spredzy**
Make this repo passing the yamllint test and add/update the tox target
to harmonize with what other components within `ansible` namespace are
doing.

Namely:

  * `ansible/ansible-runner`
  * `ansible/awx`
  * `ansible-network/ansible_collections.vyos.vyos`
  * `ansible/workshops`

Signed-off-by: Yanis Guenane <yguenane@redhat.com>

### CODE REVIEW

**Comment by ironfroggy** [comment]
This is a great start to creating some consistency in the CI and QE processes across everything, I'm all for it. 👍

File: tox.ini [hunk 1]
```diff
@@ -13,9 +13,11 @@ whitelist_externals =
     bash
     echo
 
-[testenv:flake8]
-deps = flake8
-commands = flake8 galaxy_api tests
+[testenv:linters]
+commands =
+    pipenv install --dev
```
Comment (cutwater) [inline tox.ini]: Full set of dependencies and dev dependencies is unnecessary to run `flake8` and `yamllint` tools.

Comment (Spredzy) [reply tox.ini]: @cutwater change amended. Thanks for comment.

**Comment by Spredzy** [comment]
@cutwater up

**Pull request merged**